middleware/error-handler/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
This looks to be copying the pattern from DCS. I implemented the pattern incorrectly. The error handler is meant to be a standalone module. Should either put this back in to app.js or implement the pattern as per the link in the comments.